Src/FluentAssertions/Types/TypeSelector.cs

```diff
@@ -183,6 +183,26 @@ public TypeSelector ThatAreNotClasses()
         return this;
     }
 
+    /// <summary>
+    /// Determines wether the type is sealed
```

Comment:
🔧 It doesn't determine where a type is sealed, but returns the types that are sealed.